js/ui/toast.js

```diff
@@ -217,6 +183,10 @@ var Toast = Overlay.inherit({
                     */
                     closeOnOutsideClick: true,
 
+                    /**
+                    * @name dxToastOptions.width
+                    * @default 'auto' @for Android
+                    */
```

Comment:
Is there any reason for the new doc comment here?

Reply:
I am removing doc comment in default options for windows device. But the part for Android is still actual.
```
                    /**
                    * @name dxToastOptions.width
                    * @default 'auto' @for Android|Windows_10_Mobile
                    */
```

Reply:
Oh, got it now. Thanks for the clarification.